### ISSUE TITLE
Impaired synaptic physiology 

### ISSUE THREAD

**Issue by sarahstofel** [obophenotype/human-phenotype-ontology#7055]
**Preferred term label:**

Impaired synaptic physiology

**Synonyms**

Synaptic dysfunction

**Definition (free text, please give PubMed ID)**
Any functional impairment of a synapse. 
PMID:30439458

**Parent term (use hpo.jax.org/app)**
abnormal synaptic physiology 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 104300

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0003-2307-1226

**Comment by nicolevasilevsky** [comment]
Hi @sarahstofel - thanks for creating these tickets. One small minor thing - could you please write the labels with capital first letters (eg Impaired synaptic physiology) and add periods to the definitions (Any functional impairment of a synapse.)

Thank you!

**Comment by pnrobinson** [comment]
I am not convinced that this is a good HPO term -- looking at OMIM:104300, there abnormal phenotypes are derived from mouse models and there is no clear clinical measurement of synaptic function.
The scope of the HPO is related to observations that are used clinically, either in routine care or translational research settings. It does not include inferences about the pathophysiology, which is what I think this is.

Here is an example of a synaptic term that is related to measurements.
Abnormal synaptic transmission HP:0012535

Please reopen another issue if there is a clear measurement of synaptic function that is missing from HPO.
